ResidentsSocialCarePlatformApi/V1/Infrastructure/PersonalRelationshipsView.cs

```diff
@@ -0,0 +1,22 @@
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace ResidentsSocialCarePlatformApi.V1.Infrastructure
+{
+    public class PersonalRelationshipsView
```

Comment:
This might require Table attribute even though it's been setup as a view. I'm not sure how that works in EF Core, but worth keeping an eye on it in case you have problems fetching data